proto/buf.yaml

```diff
@@ -4,10 +4,11 @@
 # This module represents the "proto" root found in
 # the previous configuration.
 version: v1
-name: buf.build/cosmos/cosmos-sdk
+name: buf.build/crypto-org-chain/cronos
 deps:
-  - buf.build/cosmos/cosmos-proto
-  - buf.build/cosmos/gogo-proto
+  - buf.build/cosmos/cosmos-sdk:8cb30a2c4de74dc9bd8d260b1e75e176
```

Comment:
how do we decides and update the commit hashes? is there a ref name like release branch name?

Reply:
can update [tag](https://buf.build/cosmos/cosmos-sdk/commits/main) based on sdk commit, but we can't use for ibc (available > v6) and ethermint (only contains main)